pavelib/tests/test_quality.py

```diff
@@ -0,0 +1,194 @@
+"""
+Tests for paver quality tasks
+"""
+import os
+import tempfile
+import unittest
+
+import paver.tasks
+from paver.easy import call_task
+from ddt import data, ddt, file_data, unpack
+from mock import MagicMock, Mock, patch
+
+import pavelib.quality
+
+
+@ddt
+class TestPaverQualityViolations(unittest.TestCase):
+    """
+    For testing the paver violations-counting tasks
+    """
+    def setUp(self):
+        super(TestPaverQualityViolations, self).setUp()
+        self.f = tempfile.NamedTemporaryFile(delete=False)
+        self.f.close()
+        self.addCleanup(os.remove, self.f.name)
+
+    def test_pylint_parser_other_string(self):
```

Comment:
test name is not readable for me, can we change it something like test_pylint_parser_checks_formatting if that's what it's purpose is ?